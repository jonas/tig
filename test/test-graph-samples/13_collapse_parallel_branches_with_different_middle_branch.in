commit A B C D E F
    Commit A - merge B, C, D, E, and F
commit D I
    Commit D - after I
commit E J
    Commit E - after J
commit F I
    Commit F - after I
commit B C
    Commit B - after C
commit C G
    Commit C - after G
commit G H
    Commit G - after H
commit H I
    Commit H - after I
commit I J
    Commit I - after J
commit J
    Commit J
