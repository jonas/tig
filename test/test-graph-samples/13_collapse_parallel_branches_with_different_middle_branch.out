●─┬─┬─┬─╮ Commit A - merge B, C, D, E, and F
│ │ ● │ │ Commit D - after I
│ │ │ ● │ Commit E - after J
│ │ │ │ ● Commit F - after I
● │ │ │ │ Commit B - after C
●─╯ │ │ │ Commit C - after G
● ╭─╯ │ │ Commit G - after H
● │ ╭─╯ │ Commit H - after I
●─┴─│───╯ Commit I - after J
◎───╯ Commit J
